tests/pbft_manager_test.cpp

```diff
@@ -646,7 +646,7 @@ TEST_F(PbftManagerWithDagCreation, initial_dag) {
   });
 }
 
-TEST_F(PbftManagerWithDagCreation, dag_generation) {
+TEST_F(PbftManagerWithDagCreation, DISABLED_dag_generation) {
```

Comment:
https://github.com/Taraxa-project/taraxa-node/pull/1909/files#r937749615

Reply:
Not sure why I was disabling this, but I just forgot to remove this DISABLED_. It works fine